app/services/save_practice_service.rb

```diff
@@ -0,0 +1,104 @@
+class SavePracticeService
+  include CropperUtils
+
+  def initialize(params)
+    @practice = params[:practice]
+    @practice_params = params[:practice_params]
+    @avatars = ['practice_creators', 'va_employees']
+    @attachments = ['impact_photos', 'additional_documents']
+  end
+
+  def save_practice
+    updated = @practice.update(@practice_params)
+
+    update_practice_partner_practices
+    update_department_practices
+    remove_attachments
+    manipulate_avatars
+    remove_main_display_image
+    crop_main_display_image
```

Comment:
if any of these fail, what happens?
perhaps we should wrap this in a `rescue` and return false if something happens and tell the user

Reply:
that's a great idea 😅 will do!